src/antsibull_changelog/constants.py

```diff
@@ -0,0 +1,16 @@
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2020, Ansible Project
+
+
+"""
+Return codes used in cli.py and test files.
+"""
+
+SUCCESS = 0  # Success
+UNHANDLED_ERROR = 1  # Unhandled error.  See the Traceback for more information.
+BAD_CLI_ARG = 2  # There was a problem with the command line arguments.
+INVALID_FRAGMENT = 3  # Found invalid changelog fragments.
+OLD_PYTHON = 4  # Needs to be run on a newer version of Python.
+COMMAND_FAILED = 5  # Problem occurred which prevented the execution of the command.
```

Comment:
If this file is only about return codes, it shouldn't be called `constants.py`, but `return_codes.py` or something like that. I would rather use this file for generic constants, and distinguish the return code constants from generic ones by adding some prefix like `RC_`. What do you think?
```suggestion
"""
Constants used by antsibull-changelog.
"""


# Return codes used in cli.py and test files.
RC_SUCCESS = 0  # Success
RC_UNHANDLED_ERROR = 1  # Unhandled error.  See the Traceback for more information.
RC_BAD_CLI_ARG = 2  # There was a problem with the command line arguments.
RC_INVALID_FRAGMENT = 3  # Found invalid changelog fragments.
RC_OLD_PYTHON = 4  # Needs to be run on a newer version of Python.
RC_COMMAND_FAILED = 5  # Problem occurred which prevented the execution of the command.
```

Reply:
Yes that sounds like a good idea. I'll update accordingly.. thanks!

Do you know why the codecove test is failing below?

Reply:
I'm not sure (also it no longer does), it often fails when for some reason it didn't get all coverage info yet, and sometimes also for unknown reasons much later on... Sometimes I'm wondering whether it's doing a count of all lines of text in the repo it can find and compares it to the 'covered' ones and then complains if the ratio decreases, even if the lines of text added have nothing to do with code... :shrug: